MarsRover.Tests/ControllerShould.cs

```diff
@@ -25,48 +25,17 @@ public void Setup_ReturnsRoverLocationAndDirection_GivenValidMapAndRoverData()
                 .Returns("2,0");
             var output = new StubOutput();
             var mockMapInput = new Mock<IMapInput>();
-            mockMapInput.Setup(_ => _.Read("TestFiles/validfile1.txt"));
-            var map = SetMap();
-            var rover = new Rover(Direction.North, 2, 0);
+            mockMapInput.Setup(_ => _.Read("TestFiles/validFile1.txt"))

```

Comment:
i recommend either using the real map input or making your own stub to make use of the contents of your text files